src/test/java/uk/gov/ons/ctp/response/collection/exercise/endpoint/CollectionExerciseClient.java

```diff
@@ -280,4 +280,31 @@ public void createCollectionExerciseEvent(EventDTO event) {
               event.getCollectionExerciseId(), event.getTag(), response.getStatus()));
     }
   }
+
+  void sampleSummaryReadiness(final UUID sampleSummaryId) throws CTPException {
+    sampleSummaryReady(sampleSummaryId);
+  }
+
+  private int sampleSummaryReady(final UUID sampleSummaryId) throws CTPException {
+    try {
+      JSONObject jsonPayload = new JSONObject();
```

Comment:
vulnerability on JSONObject
https://devhub.checkmarx.com/cve-details/CVE-2022-45689/
https://devhub.checkmarx.com/cve-details/CVE-2022-45690/

Reply:
Out of scope for this card